cmd/metal-api/main.go

```diff
@@ -888,15 +891,16 @@ func evaluateVPNConnected() error {
 		return err
 	}
 
-	ms, err := ds.ListMachines()
+	ms := metal.Machines{}
+	err = ds.SearchMachines(&datastore.MachineSearchQuery{AllocationRole: &metal.RoleFirewall}, &ms)
```

Comment:
i think we must search for all machines because there are some machines, not firewalls, which are connected, like storage in test.

Reply:
That's a good point.